### PR TITLE
Refactor: reduce amount of inline javascript

### PR DESCRIPTION
**Pull request by xi**

### CODE REVIEW

**Comment by xi** [comment]
related to #241

File: devices/templatetags/devicetags.py [hunk 1]
```diff
@@ -129,3 +130,10 @@ def as_nested_list(factvalue):
 @register.filter
 def splitstr(arg1, arg2):
     return arg1.split(arg2)
+
+
+@register.inclusion_tag('deletebutton.html')
+def deletebutton(viewname, *args):
```
Comment (octomike) [inline devices/templatetags/devicetags.py]: ah yes, I like that :)

But the template changes make the button look very differently.
2018-09-07-113012_199x249_scrot

Also, should that html go to `snippets/` ?

File: templates/devices/device_detail.html [hunk 2]
```diff
@@ -486,6 +504,7 @@ <h4>{% trans "10 last sent mails" %}</h4>
                                     <td>{{ mail.subject }}</td>
                                     <td><a href="{% url 'userprofile' mail.sent_by.pk %}">{{ mail.sent_by }}</a></td>
                                     <td><span title="{{ mail.sent_at|date:'c' }}"
+                                              data-timeago
```
Comment (octomike) [inline templates/devices/device_detail.html]: Do you know why that was previously wrapped inside a `document.ready()` ?

Comment (xi) [reply templates/devices/device_detail.html]: It still is.

File: devices/templatetags/devicetags.py (hunk 1 above)
Comment (xi) [reply devices/templatetags/devicetags.py]: done

**Pull request merged**